kernel_hardening_checker/test_engine.py

```diff
@@ -130,6 +130,8 @@ def test_simple_kconfig(self) -> None:
         config_checklist += [KconfigCheck('reason_8', 'decision_8', 'NAME_8', 'is not off')]
         config_checklist += [KconfigCheck('reason_9', 'decision_9', 'NAME_9', 'is not off')]
         config_checklist += [KconfigCheck('reason_10', 'decision_10', 'NAME_10', 'is not off')]
+        config_checklist += [KconfigCheck('reason_11', 'decision_11', 'NAME_11', '*expected_11*')]
+        config_checklist += [KconfigCheck('reason_12', 'decision_12', 'NAME_11', '*expected_12*')]
```

Comment:
Please fix a typo here.

Reply:
are you about `NAME_11` ?
this not a typo mistake, this value contain list, like `CONFIG_LSM`
that is why we have not only one value to check with `NAME_11
`

Reply:
Ok, in that case please reuse `'reason_11', 'decision_11'` for `NAME_11`.